web/site/nuxt.config.ts

```diff
@@ -2,18 +2,36 @@
 export default defineNuxtConfig({
   devtools: { enabled: false },
   ssr: true,
+
```

Comment:
The new nuxt version generates as a SPA, so now we must explicitly set what routes to prerender for nuxt content to generate correctly